sql/core/src/test/scala/org/apache/spark/sql/UrlFunctionsSuite.scala

```diff
@@ -67,6 +69,19 @@ class UrlFunctionsSuite extends QueryTest with SharedSparkSession {
         "inva lid://user:pass@host/file;param?query;p2",
         Row(null, null, null, null, null, null, null, null, null))
     }
+
+    withSQLConf(SQLConf.ANSI_ENABLED.key -> "true") {
+      val url = "inva lid://user:pass@host/file;param?query;p2"
+      checkError(
+        exception = intercept[SparkException] {
+               sql(s"SELECT parse_url('$url', 'HOST')").collect()
```

Comment:
Could you fix indentation here, see https://github.com/databricks/scala-style-guide#spacing-and-indentation